bee-node/src/plugins/version_checker.rs

```diff
@@ -30,8 +34,14 @@ impl<N: Node> Worker<N> for VersionChecker {
                 IntervalStream::new(interval(Duration::from_secs(CHECK_INTERVAL_SEC))),
             );
 
+            let client = reqwest::Client::new();
+
             while ticker.next().await.is_some() {
-                // TODO
+                match is_latest(&client).await {
+                    Ok(false) => warn!("A new version has been released. Please update the node."),
```

Comment:
Maybe include the URL of the release: https://github.com/iotaledger/bee/releases